components/domains/registerSummary.tsx

```diff
@@ -87,6 +113,15 @@ const RegisterSummary: FunctionComponent<RegisterSummaryProps> = ({
         )}$ worth of ${displayedCurrency} for Swiss VAT)`
       : "";
 
+    if (isUpselled && discountedPrice) {
+      return displayDiscountedPrice(
+        numberToFixedString(Number(gweiToEth(registrationPrice)), 3),
+        numberToFixedString(Number(gweiToEth(discountedPrice)), 3)
+          .concat(` ${displayedCurrency} `)
+          .concat(recurrence),
```

Comment:
this `.concat` could be managed directly by displaying discounted prices and adding the currency/renewal as an argument. 

I find it cleaner to read :)

Reply:
I didn't understand, you mean the currency is an argument of `displayTokenPrice` ?

Reply:
Yes